pkg/gatherers/clusterconfig/openshift_logging.go

```diff
@@ -0,0 +1,50 @@
+package clusterconfig
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+)
+
+// GatherOpenshiftLogging collects `logging.openshift.io` resources
+// from Openshift Logging Stack.
+//
+// * Location in archive: config/logging/<namespace>/<name>.json
+// * Since versions:
+//   * 4.9+
+func (g *Gatherer) GatherOpenshiftLogging(ctx context.Context) ([]record.Record, []error) {
+	gatherDynamicClient, err := dynamic.NewForConfig(g.gatherKubeConfig)
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	return gatherOpenshiftLogging(ctx, gatherDynamicClient)
+}
+
+func gatherOpenshiftLogging(ctx context.Context, dynamicClient dynamic.Interface) ([]record.Record, []error) {
+	elasticsearchResource := schema.GroupVersionResource{Group: "logging.openshift.io", Version: "v1", Resource: "clusterloggings"}
```

Comment:
Why calling it `elasticsearchResource` ? The resource doesn't have anything in common with ES or does it?

Reply:
We have the GVRs in the `const.go` to have it in one place.